bindings/go/tests/utils.go

```diff
@@ -70,22 +80,15 @@ func memoryGet(entity *TestEntity) []byte {
 	} else {
 		createSelfEp(entity)
 		recvMem := AllocateNativeMemory(memAttr.Length)
-		recvRequest, _ := entity.worker.RecvTagNonBlocking(recvMem, memAttr.Length, selfEpTag, selfEpTag, &UcpRequestParams{
+		recvRequestParams := &UcpRequestParams{
```

Comment:
maybe we can move createSelfEp and UcpRequestParams inside sendRecv?
pass sendMemTypeand recvMemType as parameters to sendRecv